internal/cephfs/nodeserver.go

```diff
@@ -326,6 +333,46 @@ func (ns *NodeServer) NodeStageVolume(
 	return &csi.NodeStageVolumeResponse{}, nil
 }
 
+// setClientAddress extracts the client IP address and stores it in the subvolume metadata.
+// The connection.GetAddrs() method returns an address in the format "10.244.0.1:0/2686266785".
+// We parse this to extract just the IP address portion (e.g., "10.244.0.1") and store in metadata.
+// If '--enable-fencing' flag is set to false in CSI driver configuration, this function does nothing.
+func setClientAddress(
```

Comment:
Make this as method for ns, so that we dont need to pass the NodeID and IsFencingEnabled